mne/time_frequency/psd.py

```diff
@@ -170,11 +375,7 @@ def compute_epochs_psd(epochs, picks=None, fmin=0, fmax=np.inf, tmin=None,
 
     data = epochs.get_data()[:, picks][..., time_mask]
     if proj:
-        proj, _ = make_projector_info(epochs.info)
-        if picks is not None:
-            data = np.dot(proj[picks][:, picks], data)
-        else:
-            data = np.dot(proj, data)
+        epochs = epochs.apply_proj()
```

Comment:
good points - OK I will try it with inst.copy().apply_proj(). I didn't realize that it operated in place since it returned an object as well.